.github/workflows/countline.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
This file does not comply with `pylint`, `pydocstyle`, `pycodestyle` and `pep8` . Please run those commands against it and fix.

Reply:
pep8 has been renamed to pycodestyle

```
/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/pep8.py:2123: UserWarning: 

pep8 has been renamed to pycodestyle (GitHub issue #466)
Use of the pep8 tool will be removed in a future release.
Please install and use `pycodestyle` instead.

$ pip install pycodestyle
$ pycodestyle ...
```